apiroutes.go

```diff
@@ -353,33 +356,21 @@ func (c *appContext) getBlockRangeSummary(w http.ResponseWriter, r *http.Request
 
 	// TODO: check that we have all in range
 
-	// N := idx - idx0 + 1
-	// summaries := make([]*apitypes.BlockDataBasic, 0, N)
-	// for i := idx0; i <= idx; i++ {
-	// 	summaries = append(summaries, c.BlockData.GetSummary(i))
-	// }
-
-	// writeJSON(w, summaries, c.getIndentQuery(r))
-
-	w.Header().Set("Content-Type", "application/json; charset=utf-8")
-	encoder := json.NewEncoder(w)
-	indent := c.getIndentQuery(r)
-	prefix, newline := indent, ""
-	encoder.SetIndent(prefix, indent)
-	if indent != "" {
-		newline = "\n"
-	}
-	fmt.Fprintf(w, "[%s%s", newline, prefix)
+	N := idx - idx0 + 1
+	summaries := make([]*apitypes.BlockDataBasic, 0, N)
 	for i := idx0; i <= idx; i++ {
-		// TODO: deal with the extra newline from Encode, if needed
-		if err := encoder.Encode(c.BlockData.GetSummary(i)); err != nil {
-			apiLog.Infof("JSON encode error: %v", err)
-		}
-		if i != idx {
-			fmt.Fprintf(w, ",%s%s", newline, prefix)
-		}
+		summaries = append(summaries, c.BlockData.GetSummary(i))
 	}
-	fmt.Fprintf(w, "]")
```

Comment:
looks like this PR is undoing this commit from yesterday:
https://github.com/dcrdata/dcrdata/commit/43540ec3f70ed3f6e7597e12e788cf20405609fc